### PR TITLE
[Bug Fix] Fix estimator python validation method argument type bug

### PR DESCRIPTION
**Pull request by yangw1234**
py4j will translate python list to java.util.ArrayList not scala's Array.

### CODE REVIEW

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/estimator/python/PythonEstimator.scala [hunk 1]
```diff
@@ -72,7 +72,8 @@ class PythonEstimator[T: ClassTag](implicit ev: TensorNumeric[T]) extends Python
                      endTrigger: Trigger = null,
                      checkPointTrigger: Trigger = null,
                      validationSet: FeatureSet[Sample[T]] = null,
-                     validationMethod: Array[ValidationMethod[T]] = null, batchSize: Int)
+                     validationMethod: JList[ValidationMethod[T]] = null,
```
Comment (qiuxin2012) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/estimator/python/PythonEstimator.scala]: @cyita Seems you should use JList here. Not Array.

Comment (cyita) [reply zoo/src/main/scala/com/intel/analytics/zoo/pipeline/estimator/python/PythonEstimator.scala]: Yes.

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/estimator/python/PythonEstimator.scala [hunk 2]
```diff
@@ -84,7 +85,7 @@ class PythonEstimator[T: ClassTag](implicit ev: TensorNumeric[T]) extends Python
 
     estimator.train(trainMiniBatch, criterion,
       Some(endTrigger), Some(checkPointTrigger),
-      validationMiniBatch, validationMethod)
+      validationMiniBatch, validationMethod.asScala.toArray)
```
Comment (cyita) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/estimator/python/PythonEstimator.scala]: If validationMethod == null, .asScala may raise an error.

Comment (yangw1234) [reply zoo/src/main/scala/com/intel/analytics/zoo/pipeline/estimator/python/PythonEstimator.scala]: changed

**Comment by yangw1234** [comment]
jenkins pass http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/2729/

**Pull request merged**